native/cocos/bindings/event/EventDispatcher.cpp

```diff
@@ -333,6 +342,26 @@ void EventDispatcher::dispatchControllerEvent(const ControllerEvent &controllerE
     EventDispatcher::doDispatchJsEvent(eventName, args);
 }
 
+void EventDispatcher::dispatchControllerChangeEvent(const ControllerChangeEvent &changeEvent) {
+    se::AutoHandleScope scope;
+    if (!jsControllerChangeEventArray) {
+        jsControllerChangeEventArray = se::Object::createArrayObject(0);
+        jsControllerChangeEventArray->root();
+    }
+
+    const char *eventName = "onControllerChange";
```

Comment:
What will trigger this event?

Reply:
The connection/disconnection of controllers